docs/resources.md

```diff
@@ -642,6 +642,18 @@ spec:
           $(inputs.resources.testCluster.name) apply -f /workspace/service.yaml'
 ```
 
+To use the `cluster` resource with Google Container Engine, you should use the `cadata` authentication
+mechanism.
+
+To determine the caData and endpoint, you can use the following `gcloud` commands:
+
+```shell
+gcloud container clusters describe <cluster-name> --format='value(masterAuth.clusterCaCertificate)'
+gcloud container clusters describe <cluster-name> --format='value(endpoint)'
+```
+
+Then, create a secret with this information and use it as described above.
```

Comment:
Can you distill this advice into a single easily-copypasteable script?

```
ENDPOINT=$(gcloud container clusters describe ${CLUSTER_NAME} --format='value=(endpoint)')
kubectl create sercret generic ${SECRET_NAME} \
  --from-literal=endpoint=${ENDPOINT} \
 ...
```

Reply:
I'll do that for the secret, but the endpoint actually goes into the resource directly under url. I'll show that with a smaller example I guess.

Reply:
How's this? I stopped short of a full cat << EOF to create the secret and resource.